09-mapping.Rmd

```diff
@@ -277,9 +297,11 @@ many_palette_plotter(c("PRGn", "YlGn", "Set2"), 7,
 <!-- how to reverse palettes with - -->
 <!-- references -->
 
+<!-- This is quite an abrupt change of topics and does not connect well with the previous paragraphs on palette options. -->
 The **tmap** package offers many methods for specifying how variables are plotted on the map, in addition to setting `breaks` manually as we have already seen in this subsection.
```

Comment:
This is quite an abrupt change of topics and does not connect well with the previous paragraphs on palette options.